uspv/init.go

```diff
@@ -137,7 +120,42 @@ func (s *SPVCon) Connect(remoteNode string) error {
 		return err
 	}
 	s.WBytes += uint64(n)
+	return nil
+}
+
+// Connect dials out and connects to full nodes.
+func (s *SPVCon) Connect(remoteNode string) error {
+	var err error
+	var listOfNodes []string
+	if lnutil.YupString(remoteNode) {
+		// if remoteNode is "yes" but no IP specified, use DNS seed
+		listOfNodes, err = s.GetListOfNodes()
+		if err != nil {
+			return err
+		}
+	} else { // else connect to user-specified node
+		if !strings.Contains(remoteNode, ":") {
+			remoteNode = remoteNode + ":" + s.Param.DefaultPort
+		}
+		listOfNodes = append(listOfNodes, remoteNode)
```

Comment:
Does the same thing, but `listOfNodes = []string{remoteNode}` looks nicer.

Reply:
yeah, looks nicer.